easybuild/easyconfigs/m/MONAI/MONAI-1.3.0-foss-2023a.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'MONAI'
+version = '1.3.0'
+
+homepage = 'https://monai.io/'
+description = """
+MONAI is a PyTorch-based, open-source framework for deep learning in healthcare
+imaging, part of PyTorch Ecosystem.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+github_account = 'Project-MONAI'
+
+dependencies = [
+    ('Python', '3.11.3'),
```

Comment:
`Python` only provides a minimal install. We should also add `Python-bundle-PyPI`. And I would also add SciPy-bundle, although it is already pulled through `PyTorch`.